api/v1alpha1/suite_test.go

```diff
@@ -1,3 +1,5 @@
+// +build integration
```

Comment:
can we keep these as part of the unit tests? This just tests that you can add the schema to the api